skbio/io/registry.py

```diff
@@ -558,11 +558,25 @@ def _init_reader(self, file, fmt, into, verify, kwargs, io_kwargs):
 
         reader = self.get_reader(fmt, into)
         if reader is None:
+            readers = self._get_possible_readers(fmt)
+            possible_intos = [r.__name__ for r in readers] if readers else None
+            if possible_intos:
+                message = """Possible values for `into` include: %s
+                          """ % ', '.join(possible_intos)
```

Comment:
Not a huge fan of using triple-quotes here. Could you change to 
```python
message = ("possible values for `into include: %s" 
           % ', '.join(possible_intos))
```
That would match most of our code-base a little better.